web/cobrands/sass/_base.scss

```diff
@@ -2392,6 +2392,39 @@ a#geolocate_link.loading, .btn--geolocate.loading {
     vertical-align: -2px;
 }
 
+$nicetable-hover-background: rgba($primary, 0.5);
+
+.nicetable {
+    width: 100%;
+    margin-bottom: 1.5em;
+
+    td, th {
```

Comment:
I miss the alternate colouring :)

Reply:
😢 🌹 👋

Reply:
Now includes a `.stripe` class you can add to the table to enable alternate row striping.

We used to do it via a `class="a"` on alternate rows, but that makes it tricky to add padding to the non-`a` siblings. And striping seems like the sort of enhancement that we can happily add for modern browsers (IE9 and up) through `:nth-child()` and leave out for IE8 and lower.